components/server/src/database/measurements.py

```diff
@@ -51,10 +53,12 @@ def recent_measurements_by_metric_uuid(database: Database, max_iso_timestamp: st
     # however, there is no test anymore covering that endpoint, which means incomplete coverage on this if statement
     if metric_uuids is not None:  # pragma: no cover
         measurement_filter["metric_uuid"] = {"$in": metric_uuids}
+    projection = {"_id": False, "metric_uuid": True, "start": True, "end": True}
+    projection.update({scale + ".value": True for scale in data_model.get("scales", {}).keys()})
```

Comment:
Because the actual measured value is stored inside the dictionary with the scale name as key:
{
    ...
    <scale>: {
        "value":  5
    }
    ...
}